### PR TITLE
add some new swords like Kriegsmesser and Falx

### PR DESCRIPTION
**Pull request by adudewithapipe**
#### Summary
Content "A couple of new swords for your entertainment."

#### Purpose of change
Mostly me practicing JSon and adding more exotic swords :)

#### Describe the solution
I added 3 new swords, pretty simple buisiness overall.

#### Describe alternatives you've considered
Not adding the cool swords.

#### Testing
I played it and it seems to be functional.

#### Additional context
Swords are cool dawg.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
You may want to change the PR title.

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -1554,6 +1569,72 @@
...
     "weapon_category": [ "MEDIEVAL_SWORDS", "SHORT_SWORDS" ],
     "category": "weapons"
+  },
+    {
+    "id": "shortsword",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": { "str": "shortsword" },
+    "description": "A sword, larger than a dagger, but shorter than an Arming Sword, and the perfect length for a sidearm.  Despite its diminuative size compared to other swords, it's still a deadly weapon.",
```
Comment (TheShadowFerret) [inline data/json/items/melee/swords_and_blades.json:1579]:
```suggestion
    "description": "A sword, larger than a dagger, but shorter than an arming sword, and the perfect length for a sidearm.  Despite its diminutive size compared to other swords, it's still a deadly weapon.",
```

File: data/json/items/melee/swords_and_blades.json [hunk 2]
```diff
@@ -1554,6 +1569,72 @@
...
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 10 ] ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS" ]
+  },
+      {
+    "id": "kriegsmesser",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": { "str": "kriegsmesser" },
+    "description": "A single edged hand and a half sword of German design, similar to a machete with a handguard.  Cheaper and lighter than a longsword, but a devastating cutting weapon nonetheless.",
```
Comment (TheShadowFerret) [inline data/json/items/melee/swords_and_blades.json:1601]:
```suggestion
    "description": "A single-edged hand-and-a-half-sword of German design, similar to a machete with a handguard.  Cheaper and lighter than a longsword, but a devastating cutting weapon nonetheless.",
```

File: data/json/items/melee/swords_and_blades.json [hunk 3]
```diff
@@ -1554,6 +1569,72 @@
...
+    "techniques": [ "WBLOCK_2" , "RAPID" ],
+    "weight": "900 g",
+    "volume": "1500 ml",
+    "longest_side": "50 cm",
+    "bashing": 3,
+    "cutting": 28,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "good" },
+    "category": "weapons",
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 10 ] ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS" ]
```
Comment (TheShadowFerret) [inline data/json/items/melee/swords_and_blades.json:1593]:
```suggestion
    "weapon_category": [ "MEDIEVAL_SWORDS", "SHORT_SWORDS" ]
```
I'm having a big think 🤔

**Review by TheShadowFerret — COMMENTED** [review]
I read the title at first and thought it was an upgraded sharpened pipe using a tee or cross fitting as a crossguard 😄

**Comment by catdach** [comment]
Hmm... I have mixed feelings about adding a generic "short sword" to the game. AFAIK what most people think of as a medieval short sword is almost always an arming-sword. I'm not personally aware of many medieval swords being shorter than that, but I guess it's possible. The Xiphos would be a compairable weapon. Maybe it shouldn't have "rapid"? 🤔

I know nothing about the other swords, but they seem pretty cool 👍

**Review by Aerin-of-the-Toast — APPROVED** [review]

**Pull request merged**